### DISCUSSION TITLE
Add more examples / SDKs : FastAPI, Rails, SvelteKit, PhoenixFramework

### DISCUSSION THREAD

**Discussion by nileshtrivedi** [zitadel/zitadel#8239] [Ideas]
I noticed that many examples/guides are missing for many popular frameworks for many languages: FastAPI (Python), Rails (Ruby), SvelteKit (JS), Phoenix (Elixir). Having them would make it easier for developers to adopt Zitadel.

**Comment by hifabienne** [comment]
Hei @nileshtrivedi 
We do have an open discussion about which frameworks and languages we should provide next here: https://github.com/zitadel/zitadel/discussions/7376
We do not yet have a timeline about when we will implement them
